### DISCUSSION TITLE
Hi, I want to know SnapshotConnection.openRemote Application scenarios of function,

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#582] [Q&A] [answered]
Hi, I want to know SnapshotConnection.openRemote Application scenarios of function
Thanks.

**Comment by ramanujam-raman-zz** [comment, marked as answer]
The method is meant to be used if you have written a dedicated backend that hosts a snapshot of the iModel. This however is not done in any of the Bentley hosted backends. 

The more typical use is SnapshotConnection.openFile(), and this opens up a local copy of the snapshot.
